influxdb3/src/commands/serve.rs

```diff
@@ -378,6 +382,31 @@ pub async fn command(config: Config) -> Result<()> {
     Ok(())
 }
 
+async fn setup_telemetry_store(
+    object_store_config: &ObjectStoreConfig,
+    instance_id: Arc<str>,
+    num_cpus: usize,
+) -> Arc<TelemetryStore> {
+    let os = std::env::consts::OS;
+    let influxdb_pkg_version = env!("CARGO_PKG_VERSION");
```

Comment:
@praveen-influx - sorry didn't catch this during review, but I think it would be worth re-using from the [`influxdb3_process` workspace crate](https://github.com/influxdata/influxdb/blob/29daa8332d78498595dd8ba62e0f4cb5060c2730/influxdb3_process/src/lib.rs) for some of these environment variables.

Reply:
I can use it for the version as built [here](https://github.com/influxdata/influxdb/blob/29daa8332d78498595dd8ba62e0f4cb5060c2730/influxdb3_process/src/lib.rs#L34). 

I'll probably need other lookup for the package name `CARGO_PKG_NAME` - is it worth adding that lookup to the `influxdb3_process` crate as well?